styles.css

```diff
@@ -240,3 +240,7 @@ tr {
     margin-top: -29px;
   }
 }
+
+a {
+  word-break: break-all;
+}
```

Comment:
I think an `<article>` needs a classname such as `.post` or `.archive`. Don't like to style elements directly by DOM names.